README.rst

```diff
@@ -0,0 +1,112 @@
+#########
+ctrl_oods
+#########
+
+``ctrl_oods`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+
+.. Add a brief (few sentence) description of what this package provides.
+
+The Observatory Operations Data Service watches for files in one or more directories, and then ingests them into an LSST Butler repository.   Files are expired from the repository at specified intervals.
+
+Usage:  oods.py [[-c config][-y config] [-v]
+
+Options:
+
+.. code:: bash
+
+ -c config use specified OODS YAML configuration file
+ -y config validate YAML configuration file
```

Comment:
Do you mean you'd rather see the `-y` dropped and have it always validate the YAML (`-c` or not).   I can do that.

Reply:
What I meant was that `foo [-y arg]` is different from `foo [-y] [arg]`.  Or, perhaps more clearly, `foo [-y arg] [-x] [-z] [other_arg]` is different from `foo [-y] [-x] [-z] other_arg`.  I think it might be best to have `oods.py [-y] [CONFIG]`.